src/qt/forms/overviewpage.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>948</width>
-    <height>559</height>
+    <height>942</height>
```

Comment:
Why is this height change required?

Reply:
To accommodate the additional fields on the main status screen...

Reply:
I'll just edit it back manually. :)